runtests.py

```diff
@@ -22,5 +22,18 @@ def run_tests(*test_args):
     sys.exit(bool(failures))
 
 
+def run_swapped_tests(*test_args):
+    if not test_args:
+        test_args = ['tests']
+
+    os.environ['DJANGO_SETTINGS_MODULE'] = 'tests.settings_swapped'
+    django.setup()
+    TestRunner = get_runner(settings)
+    test_runner = TestRunner()
+    failures = test_runner.run_tests(test_args)
+    sys.exit(bool(failures))
```

Comment:
Isn't easier if this is exactly the same tests with different settings, to move this to pytest and run there 2 tests suites by supplying them settings as a parameter?

Reply:
Worth a try :+1: